### PR TITLE
Add Library js-md4

### PR DESCRIPTION
**Pull request by kkHAIKE**

### CODE REVIEW

**Comment by Xuanwo** [comment]
Just need to add `package.json`, qiniu's server will fetch source automatically.

File: libs/js-md4/0.3.0/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# v0.3.0 / 2015-12-28
+
+* Added support for update hash.
+* Added support for bytes array output.
+* Added support for ArrayBuffer output.
+
+# v0.2.0 / 2015-12-01
```
Comment (Xuanwo) [inline libs/js-md4/0.3.0/CHANGELOG.md]: Remove all source like these, just keep `package.json`. for example: https://github.com/staticfile/static/pull/340/files

**Comment by kkHAIKE** [comment]
Confusing,it's seem new rule from away [How to Sumbit A Lib](https://www.staticfile.org/about_en.html)

**Comment by Xuanwo** [comment]
Well, both way is OK, but let server do the fetch job will be better.
Only difference is you or the server download the source file.

**Comment by Xuanwo** [comment]
And we will update the README sooner or later.

**Pull request closed without merging**